src/get-started/test-service.md

```diff
@@ -15,7 +15,7 @@ Correspondingly,
 service testing could be performed with the help
 of [integration tests][integration-testing],
 in which the service is treated like a black or gray box. In the case of
-the cryptocurrency service, which we created in the previous tutorial, it
+the demo service, which we created in the previous tutorial, it
```

Comment:
I would substitute "previous" for the exact tutorial name, i.e. 
> which we created in the "How to Create a Service" tutorial,

because the number of tutorials grows and it may cause confusion